Webex Connect Flows/v3.3/Template/README.MD

```diff
@@ -0,0 +1,57 @@
+# Media Specific Workflows
+
+## Workflows Overview
+The following workflow templates are included :
+
+**Media Specific Workflows** : These flows need to be imported once per asset of the specific channel.
+* **Facebook Inbound Flow** - This workflow will be triggered for every inbound customer message received over the integrated Facebook page.
+* **Live Chat Inbound Flow** - This workflow will be triggered for every inbound customer message received over the configured customer chat widget, and the widget has a live chat form configured.
+* **Live Chat Inbound Flow Without Form** - This workflow will be triggered for every inbound customer message received over the configured customer chat widget and the widget doesn't have a live chat form configured.
+* **SMS Inbound Flow** - This workflow will be triggered for every inbound customer sms received over the integrated SMS phone number.
+* **Email Inbound Flow** - This workflow will be triggered for every inbound customer email received on the integrated email account.
+* **Whatsapp Inbound Flow** - This workflow will be triggered for every inbound customer message received over the integrated whatsapp business number.
+* **Apple Messages For Business Inbound Flow** - This workflow will be triggered for every inbound customer message over the integrated business account.
+* **Live Chat Close Flow** - This workflow will be triggered when a livechat customer has ended the chat.
+
+Note :
+- Queue selection from queue task node is mandatory for any channel flows before making the flow live.
+- Flow variables must be updated in the setting section with the correct asset details before making the flow live.
+
+## Updating Flow Variables
+
+Flow Variables that need to be updated under Custom Variables for different channels: -
+- LiveChat :- appId and liveChatDomain
+  ![Live chat Settings](../images/Settings.png)
+
+- Facebook :- FBpageid and appid
+  ![Facebook Settings](../images/Facebook.png)
+
+- Whatsapp :- WANumber and appid
+
+  ![Whatsapp Settings](../images/Whatsapp.png)
+
+- Email    :- bizemailid
+
+  ![Email Settings](../images/Email.png)
```

Comment:
can we add for apple also here for consistency ?

Reply:
For channels where asset details need not be updated, we're not calling anything out previously as well ( such as for SMS ), hence continuing the same behaviour